gpiozero/boards.py

```diff
@@ -265,7 +265,7 @@ class LEDBarGraph(LEDCollection):
     """
 
     def __init__(self, *pins, **kwargs):
-        super(LEDBarGraph, self).__init__(*pins, pwm=False)
+        super(LEDBarGraph, self).__init__(*pins, **kwargs)
         initial_value = kwargs.get('initial_value', 0)
```

Comment:
Unfortunately, while `*pins, pwm=False` is valid in Python 3, it isn't in Python 2 (only `**kwargs` can come after `*args` in Python 2 - nothing else, including keyword args with defaults). This is the sole reason we use **kwargs in such cases; to maintain Python 2 compatibility (while it's perfectly reasonable to use Python 3 only for applications at this point, libraries probably need to support Python 2 for several more years - there's still plenty of Python 2 users out there!)